api/src/main/resources/workbench.yaml

```diff
@@ -44,18 +44,6 @@ security:
 
 ## Common Path Parameters
 parameters:
-  workspaceNamespace:
```

Comment:
Can these parameters not live in both places? (I they are shared/used in both files?)

Reply:
We *could* define things in two places, but only one of them will wind up being used... if they are inconsistent this would be confusing. So IMHO it's better to define shared things in client_api.yaml and reference them in workbench.yaml. Our validation of the merged YAML ensures that everything is actually defined somewhere.

Reply:
I added a comment to the top of workbench.yaml about this stuff / what should go where.